src/controllers/controllerengine.cpp

```diff
@@ -86,12 +86,16 @@ void ControllerEngine::callFunctionOnObjects(QList<QString> scriptFunctionPrefix
         QScriptValue prefix = global.property(prefixName);
         if (!prefix.isValid() || !prefix.isObject()) {
             qWarning() << "ControllerEngine: No" << prefixName << "object in script";
+            // Throw a debug assertion if controllerDebug is enabled
```

Comment:
I think this is unneeded because
a) it's just a single line
b) this will mess with the source file/line number from DEBUG_ASSERT and required a backtrace to track down the location.

Reply:
Agreed. I didn't consider aspect b).